app/controllers/manager/venues_controller.rb

```diff
@@ -39,4 +51,14 @@ def require_user
       end
     end
 
+    def require_correct_manager
```

Comment:
For some reason this logic seems to make more sense to me.  What do you ( @TomSoucie @MasonHolland ) think:
```ruby
def require_correct_manager
  unless @venue.managers.include?(current_user.id)
  [...]
end
```